frontend/public/components/monitoring/receiver-forms/email-receiver-form.tsx

```diff
@@ -0,0 +1,297 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+
+import { SectionHeading } from '../../utils';
+import { SaveAsDefaultCheckbox } from './alert-manager-receiver-forms';
+
+const SMTP_GLOBAL_FIELDS = [
+  'from',
+  'smarthost',
+  'hello',
+  'auth_username',
+  'auth_password',
+  'auth_identity',
+  'auth_secret',
+  'require_tls',
+];
+const GLOBAL_FIELDS = [...SMTP_GLOBAL_FIELDS]; //TODO follow up PR will add advanced fields
+
+// Ex: converts 'auth_password' to 'emailAuthPassword'
+const getFormFieldName = (propName) =>
+  `email${_.map(_.split(propName, '_'), (part) => _.upperFirst(part)).join('')}`;
+const getFormValue = (formValues, propName) => _.get(formValues, getFormFieldName(propName));
+const getGlobalValue = (globals, propName) => {
+  const fldName = `${SMTP_GLOBAL_FIELDS.includes(propName) ? 'smtp' : 'email'}_${propName}`;
+  return _.get(globals, fldName);
+};
+
+export const Form = ({ globals, formValues, dispatchFormChange }) => {
+  // disable saveAsDefault if all SMTP form fields match global values
+  const disableSaveAsDefault = _.every(
+    SMTP_GLOBAL_FIELDS,
+    (propName) => getFormValue(formValues, propName) === getGlobalValue(globals, propName),
+  );
+
+  return (
+    <div data-test-id="email-receiver-form">
+      <div className="form-group">
+        <label className="control-label co-required">To Address</label>
+        <input
+          className="pf-c-form-control"
+          type="text"
+          aria-describedby="email-to-help"
+          name="emailTo"
```

Comment:
Do we still need `name`? (Here and for other inputs)

Reply:
No, removed